packages/pillarbox-playlist/src/pillarbox-playlist-menu-item.js

```diff
@@ -0,0 +1,67 @@
+import videojs from 'video.js';
+
+/**
+ * @ignore
+ * @type {typeof import('video.js/dist/types/menu/menu-item').default}
+ */
+const Button = videojs.getComponent('Button');
+
+/**
+ * Class representing a playlist menu item in the Pillarbox plugin.
+ */
+class PillarboxPlaylistMenuItem extends Button {
+  /**
+   * Creates an instance of PillarboxPlaylistMenuItem.
+   * @param {Object} player - The video.js player instance.
+   * @param {Object} options - Options for the menu item.
+   * @param {number} options.index - The index of the playlist item.
+   * @param {Object} options.item - The playlist item data.
+   * @param {Object} options.item.data - The data for the playlist item.
+   * @param {string} options.item.data.title - The title of the playlist item.
+   * @param {number} options.item.data.duration - The duration of the playlist item.
+   */
+  constructor(player, options) {
+    super(player, options);
+    this.selected(options.index === this.playlist().currentIndex);
+    this.controlText(`${this.options_.index + 1}. ${this.options_.item.data?.title} - ${videojs.formatTime(this.options_.item.data?.duration)}`);
```

Comment:
Could we remove the `index` from the `controlText` ? See my other comment below.